test/test_mathics_precedence.py

```diff
@@ -0,0 +1,606 @@
+    "SquareSupersetEqual",
+    "Subset",
+    "SubsetEqual",
+    "Superset",
+    "SupersetEqual",
+    "DoubleLeftArrow",
+    "DoubleLeftRightArrow",
+    "DoubleRightArrow",
+    "DownLeftRightVector",
+    "DownLeftTeeVector",
+    "DownLeftVector",
+    "DownLeftVectorBar",
+    "DownRightTeeVector",
+    "DownRightVector",
+    "DownRightVectorBar",
+    "LeftArrow",
+    "LeftArrowBar",
+    "LeftArrowRightArrow",
+    "LeftRightArrow",
+    "LeftRightVector",
+    "LeftTeeArrow",
+    "LeftTeeVector",
+    "LeftVector",
+    "LeftVectorBar",
+    "LowerLeftArrow",
+    "LowerRightArrow",
+    "RightArrow",
+    "RightArrowBar",
+    "RightArrowLeftArrow",
+    "RightTeeArrow",
+    "RightTeeVector",
+    "RightVector",
+    "RightVectorBar",
+    "ShortLeftArrow",
+    "ShortRightArrow",
+    "UpperLeftArrow",
+    "UpperRightArrow",
+    "DoubleVerticalBar",
+    "NotDoubleVerticalBar",
+    "VerticalBar",
+    "Equal",
+    "Greater",
+    "GreaterEqual",
+    "Less",
+    "LessEqual",
+    "GreaterSlantEqual",
+    "LessSlantEqual",
+    "SameQ",
+    "Unequal",
+    "UnsameQ",
+    "Congruent",
+    "CupCap",
+    "DotEqual",
+    "EqualTilde",
+    "Equilibrium",
+    "GreaterEqualLess",
+    "GreaterFullEqual",
+    "GreaterGreater",
+    "GreaterLess",
+    "GreaterTilde",
+    "HumpDownHump",
+    "HumpEqual",
+    "LeftTriangle",
+    "LeftTriangleBar",
+    "LeftTriangleEqual",
+    "LessEqualGreater",
+    "LessFullEqual",
+    "LessGreater",
+    "LessLess",
+    "LessTilde",
+    "NestedGreaterGreater",
+    "NestedLessLess",
+    "NotCongruent",
+    "NotCupCap",
+    "NotGreater",
+    "NotGreaterEqual",
+    "NotGreaterFullEqual",
+    "NotGreaterLess",
+    "NotGreaterTilde",
+    "NotLeftTriangle",
+    "NotLeftTriangleEqual",
+    "NotLess",
+    "NotLessEqual",
+    "NotLessFullEqual",
+    "NotLessGreater",
+    "NotLessTilde",
+    "NotPrecedes",
+    "NotPrecedesSlantEqual",
+    "NotPrecedesTilde",
+    "NotRightTriangle",
+    "NotRightTriangleEqual",
+    "NotSucceeds",
+    "NotSucceedsSlantEqual",
+    "NotSucceedsTilde",
+    "NotTilde",
+    "NotTildeEqual",
+    "NotTildeFullEqual",
+    "NotTildeTilde",
+    "Precedes",
+    "PrecedesEqual",
+    "PrecedesSlantEqual",
+    "PrecedesTilde",
+    "Proportion",
+    "Proportional",
+    "ReverseEquilibrium",
+    "RightTriangle",
+    "RightTriangleBar",
+    "RightTriangleEqual",
+    "Succeeds",
+    "SucceedsEqual",
+    "SucceedsSlantEqual",
+    "SucceedsTilde",
+    "Tilde",
+    "TildeEqual",
+    "TildeFullEqual",
+    "TildeTilde",
+    # In Mathics, the precedence of these operators is quite low.
+    #    "DirectedEdge",  # Mathics 128 , WMA 295
+    #    "UndirectedEdge", # Mathics 120, WMA 295
+    "SquareUnion",
+    "UnionPlus",
+    "Span",
+    "SquareIntersection",
+    "MinusPlus",
+    "PlusMinus",
+    "Plus",
+    "Subtract",  #  310
+    #    "Integrete", # In Mathics, this has the default precedence. In WMA, 325
```

Comment:
At some point, we need to look over again the way how the information for `Integral` and `Integrate` operators are stored in the tables..